src/constraints/constraint_ratio_unit_flow.jl

```diff
@@ -223,7 +264,9 @@ function constraint_ratio_unit_flow_indices(m::Model, ratio)
     (
         (unit=u, node1=n1, node2=n2, stochastic_path=path, t=t)
         for (u, n1, n2) in indices(ratio)
-        if !_has_simple_fix_ratio_unit_flow(n1, n2, ratio)
+        if !_has_simple_fix_ratio_unit_flow(n1, n2, ratio) || 
+            !isempty(unit_flow_op_indices(m; unit=u, node=n1, direction=d1)) || 
+            !isempty(unit_flow_op_indices(m; unit=u, node=n2, direction=d2))
```

Comment:
I think we should put the two new checks inside the `_has_simple_fix_ratio_unit_flow` function for readability.

Reply:
That's what I would have liked to have as well, but m, unit and direction were not given to `_has_simple_fix_ratio_unit_flow` so I didn't want to start changing that function too much. But if you think that's the cleaner way, let's try to update code accordingly.

Reply:
Hi @nhniina - yes I think so, because we also do the checks in some other place (when creating the variable) so it's better if we don't repeat that code. Thanks! Great work btw :)